tests/post/steps/test_service_configuration.py

```diff
@@ -199,6 +184,45 @@ def restore_csc(
     )
 
 
+@then(parsers.parse(
+    "we have an alert rule '{rule_name}' in group '{group_name}' with "
+    "'{path}' equal to '{value}'"
+))
+def check_prometheus_alert_rule(
+        prometheus_api, rule_name, group_name, path, value):
+    """Retrieve an alert rule from the Prometheus API, then
+    checks if the given path matches the value.
+    """
+
+    def _wait_prometheus_config_reload():
+        try:
+            response = prometheus_api.get_rules()
+        except utils.PrometheusApiError as exc:
+            pytest.fail(str(exc))
+
+        for group in response.get('data', {}).get('groups', []):
+            if group.get('name') == group_name:
+                for rule in group.get('rules', []):
+                    if rule.get('name') == rule_name:
+                        assert utils.get_dict_element(rule, path) == value
+                        break
+                else:
+                    continue
+                break
+        else:
+            pytest.fail(
+                "Alert rule {} in group {} not found".format(
+                    rule_name, group_name
+                )
+            )
```

Comment:
Since you've created your own helper to manage Prom API, why not give it this logic to find a specific rule? Could be more re-usable.

Reply:
And just handle comparing values in this test

Reply:
Yes, why not.